packages/frontend-v2/pages/index.tsx

```diff
@@ -1,429 +1,86 @@
 import type { NextPage } from "next";
-import { useState } from "react";
-import { SearchAPI, API } from "@graduate/api-client";
-import { toast, logger } from "../utils";
-
-const Bomb: React.FC = () => {
-  throw Error("BOOOOM!");
-};
+import { Box, Button, Flex, Heading, HStack, Image, SimpleGrid, Text, VStack } from "@chakra-ui/react";
+import { Logo } from "../components/Header/header";
 
 const Home: NextPage = () => {
-  const [isClientSideError, setIsClientSideError] = useState(false);
-
-  const [token, setToken] = useState("");
-  const [planId, setPlanId] = useState<number>();
-
-  if (isClientSideError) {
-    return <Bomb />;
-  }
-
-  // handler that calls fetchCourse from our api-client package
-  const testFetchCourse = (subject: string, classId: string) => {
-    SearchAPI.fetchCourse(subject, classId).then((res) => console.log(res));
-  };
-
-  // handler that calls searchCourses from our api-client package
-  const testSearchCourses = (
-    searchQuery: string,
-    minIndex: number,
-    maxIndex: number
-  ) => {
-    SearchAPI.searchCourses(searchQuery, minIndex, maxIndex).then((res) =>
-      console.log(res)
-    );
-  };
+  return (
+    <Box>
+      <Header />
+      <Banner />
+      <Info />      
+    </Box>
+  );
+};
 
+const Header = (): JSX.Element => {
   return (
-    <>
-      <h1>GraduateNU Landing Page:</h1>
-      <div>
-        <h2>Testing api calls on backend v2</h2>
-        <p>All responses will be logged to the console</p>
+    <Flex layerStyle="headerContainer">
+      <Logo />
+      <Button size="sm"> Sign In</Button>
+    </Flex>
+  );
+}
 
-        <div>
-          <h3>Auth Routes</h3>
-          <button
-            onClick={async () => {
-              const student = await API.auth.register({
-                fullName: "Aryan Shah",
-                email: "aryan1@gmail.com",
-                password: "aryan1234",
-                academicYear: 2019,
-                graduateYear: 2023,
-                catalogYear: 2019,
-                major: "Computer Science",
-                nuid: "000000000",
-              });
-              console.log(student);
-              setToken(student.accessToken!);
-            }}
-          >
-            Register
-          </button>
+const Banner = (): JSX.Element => {
+  return (
+    <Flex pt={{ desktop: "75px", laptop: "50px", tablet: "40px" }} pb={{ desktop: "175px", laptop: "150px", tablet: "100px"}}>
+      <HStack direction="row" spacing={{ desktop: "100px", laptop: "75px", tablet: "50px" }} justifyContent="center">
+        <Image boxSize={{ desktop: "550px", laptop: "500px", tablet: "400px" }} src="/husky.svg" alt="husky"/>
+        <Flex w="35%" flexDirection="column" alignItems="center">
+          <Box>
+            <Heading fontSize={{ desktop: "7xl", laptop: "6xl", tablet: "5xl"}} color="primary.main">
```

Comment:
the `size` prop range for chakra headings was not quite as large as i needed it to be here, so i applied the breakpoints using the `fontSize` prop instead

Reply:
good thinking!